sources/core/Stride.Core.Mathematics/Int3.cs

```diff
@@ -200,12 +190,12 @@ public int Length()
         }
 
         /// <summary>
-        /// Calculates the unrounded length of the vector.
+        /// Calculates the untruncated length of the vector.
         /// </summary>
-        /// <returns>The length of the vector unrounded.</returns>
+        /// <returns>The length of the vector untruncated.</returns>
         public float LengthUntruncated() 
         {
-            return Math.Sqrt((X * X) + (Y * Y) + (Z * Z));
+            return (float)Math.Sqrt(Length());
```

Comment:
should be `LengthSquared()`

Reply:
oops

Reply:
And actually given that Length also does the inlining manually, I shouldn't call a method but also inline.